### PR TITLE
update versions and changelog with new release of the plugins

### PR DESCRIPTION
**Pull request by tiainen**
Updates the versions and changelogs for the next release.

- eclipse: 2.6.0
- intellij: 2.7.0
- netbeans: 2.7.0

### CODE REVIEW

File: intellij/src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -1,72 +1,18 @@
...
     ]]></description>
 
     <change-notes><![CDATA[
-      <b>2.6.3</b>
+      <b>2.7.0</b>
       <ul>
-      <li>Introduce Single View project using Gluon VM</li>
+      <li>Update templates for Gluon Mobile 5</li>
       </ul>
-      <b>2.6.2</b>
```
Comment (johanvos) [inline intellij/src/main/resources/META-INF/plugin.xml]: Are older release notes stored somewhere?

Comment (tiainen) [reply intellij/src/main/resources/META-INF/plugin.xml]: Yes, we added a CHANGELOG.md file for each plugin, inside the repository itself.

These notes are only used as information on the IntelliJ plugin page. On that page, the release notes are separated per version, so it doesn't make sense to include the entire release notes each time for each release. For example, this is the page for version 2.6.3 https://plugins.jetbrains.com/plugin/7864-gluon-plugin/update/42325.

Comment (abhinayagarwal) [reply intellij/src/main/resources/META-INF/plugin.xml]: Yes, it stored in a [changelog file](https://github.com/gluonhq/ide-plugins/blob/master/intellij/CHANGELOG.md).

File: eclipse/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 ## Changelog Eclipse
 
+#### 2.6.0
+_2018-04-24
```
Comment (abhinayagarwal) [inline eclipse/CHANGELOG.md]: Date needs to be enclosed with _ on both sides :)

**Pull request merged**